tst/testlibgap/wsload.c

```diff
@@ -4,7 +4,14 @@
 #include "common.h"
 int main(int argc, char ** argv)
 {
-    GAP_Initialize(argc, argv, environ, 0L, 0L);
+    char *args[50];
+    char lpar[3] = "-L";
+    char wsname[16] = "/tmp/libgap.ws"; /* the name must match the one used in wscreate.c */
```

Comment:
Note that `/tmp` might not exist; it might also be a security issue to put a .ws there, in case the test is run on a shared machine. Finally, this will break if these tests are run by multiple jobs in parallel. However, none of these are really blockers for this PR, IMHO, but I wanted to point them out anyway.

Reply:
I added a comment in the source TODO this properly at some point.
(see the last commit)